GameSync.Api/Program.cs

```diff
@@ -111,8 +127,6 @@ public static void Main(string[] args)
 
         var app = builder.Build();
 
-        app.MapGroup("/account").MapIdentityApi<ApplicationUser>();
```

Comment:
Why you removed all endpoints from `NET.Identity`?

Reply:
We use our own endpoint to register, login, remember password, etc.